packages/modules/create-imodel/package.json

```diff
@@ -19,8 +19,7 @@
     "clean": "rimraf cjs esm"
   },
   "dependencies": {
-    "@itwin/itwinui-css": "^0.9.0",
-    "@itwin/itwinui-react": "^1.35.0"
+    "@itwin/itwinui-react": "^2.11.2"
```

Comment:
update the version here `2.11.4` to fix file upload issue i have listed.

Reply:
done